### ISSUE TITLE
[Proposal] Добавить триторам цельку на защиту.

### ISSUE THREAD

**Issue by ZER0-In** [TauCetiStation/TauCetiClassic#12094]
#### Подробное описание
Добавить триторам цельку на защиту какого-нибудь члена экипажа.
Получим различные приколы, когда у одного цель на килл, у другого на деф. И война друг с другом, либо переговоры.

**Comment by Oz1Es** [comment]
говно, зачем синдикату защищать крю шаттл

**Comment by Slavik2001** [comment]
Согласен. + уверен что будут ситуации как у мага, когда тебе надо одновременно и защитить свою цель и убить ее.

**Comment by DarkFurrSss** [comment]
> Согласен. + уверен что будут ситуации как у мага, когда тебе надо одновременно, и защитить свою цель, и убить ее.

Уверен, что это решаемая проблема. Можно разделить задачи на 
1. Убить/Защитить
2. Украсть
3. Как либо закончить раунд (Захват шаттла оставить с защитой. Пусть будет сложный момент)

**Comment by simb11** [comment]
а как по мне, цель на защиту - полная фигня, потому что ломает суть такого антагониста как ПРЕДАТЕЛЬ
на кой чёрт ПРЕДАТЕЛЮ заниматься ЗАЩИТОЙ ЭКИПАЖА

**Comment by pro100kotah** [comment]
> а как по мне, цель на защиту - полная фигня, потому что ломает суть такого антагониста как ПРЕДАТЕЛЬ на кой чёрт ПРЕДАТЕЛЮ заниматься ЗАЩИТОЙ ЭКИПАЖА

Предатель НТ. Не всего же человечества. Он на Синдикат работает. 

Синдикат исходя из лора вербует людей прямо из НТ. Поэтому даже главы могут быть триторами. Очевидно, что может появиться цель на защиту члена экипажа, который может быть выгодным для Синдиката, в связи с его взглядами, социальным статусом. Обстоятельствам в жизни, из-за которых его легко склонить к предательству.

Это аналог цельки на вербовку, только более логичный и реализуемый в наших реалиях ТК.

Но судя по тому, что у вас протект вызывает вопросы, о вербовке и переносе этих целек даже говорить нет смысла.

Не думал, что мне придется объяснять что-то настолько очевидное. Триторы это не нюкеры. Они не головорезы, у них нет цели вырезать весь экипаж.

**Comment by pro100kotah** [comment]
> говно, зачем синдикату защищать крю шаттл

bruh

**Comment by simb11** [comment]
да я те не про рп лор и тому подобную хуйню пишу
с игровой точки зрения трейтор существует для того чтобы противостоять станции и создать проблемы для неё, а не защищать зачем-то экипаж этой самый станции
